thorlcr/slave/thslavemain.cpp

```diff
@@ -383,8 +393,12 @@ int main( int argc, char *argv[]  )
         localHostToNIC(masterEp);
         setMasterPortBase(masterEp.port);
         markNodeCentral(masterEp);
+
+        mySlaveNum = globals->getPropInt("@SLAVENUM", NotFound);
         if (RegisterSelf(masterEp))
         {
+            startSlaveLog();
```

Comment:
yeah maybe, it's done here because slaveNum not known until this point in cloud setup, but is known before in bare metal - so could.
I'll change so as early as possible.
There's not much tracing before this, but would be better to capture it.